frontend/scripts/buildCache.ts

```diff
@@ -1,24 +1,37 @@
 /**
  * Codegens API typings
  */
+import * as S3 from 'aws-sdk/clients/s3'
 import Axios from 'axios'
 import 'isomorphic-fetch'
 import { seq } from 'promise-seq'
 import { CONFIG } from '../src/config/config'
 import { DATASETS } from '../src/constants/datasets'
 import { Method } from '../src/constants/datatypes'
 import {
-  getAdequacies, getCensusData, getRepresentativePoints
+  getAdequacies, getCensusData, getRepresentativePoints, getStaticAdequacyUrl, getStaticDemographicsUrl, getStaticRPUrl
 } from '../src/services/api'
 import { safeDatasetHint } from '../src/utils/formatters'
+
 const METHODS: Method[] = ['driving_time', 'straight_line']
 
+const buildCache = true
+
+/**
+* To push files to S3, you will need 'write' permissions for the bucket you are
+* trying to push the files to. This script uses your default $AWS_PROFILE.
+* WARNING - Please note that file with the same name will be overriden.
+*/
+const s3Bucket = 'encompass-public-data'
+const s3 = new S3()
+const uploadToS3 = true
```

Comment:
philip's question still stands

Reply:
It's a script so it shouldnt be too linked to the config IMO

Reply:
then why have it as an option? can we pass it in as a param to the script? I don't like this

Reply:
I think it is fine as is for now, since S3 requires credentials access anyways, but I will create a ticket to add params to this script